circle.yml

```diff
@@ -15,7 +15,12 @@ machine:
 dependencies:
     pre:
         - echo y | android update sdk --no-ui --all --filter "tools"
-        - echo y | android update sdk --no-ui --all --filter "build-tools-25.0.0"
+        - echo y | android update sdk --no-ui --all --filter "build-tools-25.2.3"
+        - echo y | android update sdk --no-ui --all --filter "extra-android-m2repository"
+        - echo y | android update sdk --no-ui --all --filter "extra-android-support"
+        - echo y | android update sdk --no-ui --all --filter "extra-google-google_play_services"
```

Comment:
Yeah it won't add any issue, but also I don't want to add stuff if we're not really using it. Anyways, is your call.